internal/resources/payments/deployments.go

```diff
@@ -120,8 +215,12 @@ func createFullDeployment(ctx core.Context, stack *v1beta1.Stack,
 							Args:          []string{"serve"},
 							Env:           env,
 							Image:         image,
-							LivenessProbe: applications.DefaultLiveness("http", applications.WithProbePath("/_health")),
+							LivenessProbe: applications.DefaultLiveness("http", appOpts),
 							Ports:         []v1.ContainerPort{applications.StandardHTTPPort()},
+							// TODO(polo): only for v3
+							SecurityContext: &v1.SecurityContext{
+								ReadOnlyRootFilesystem: pointer.For(false),
```

Comment:
Why?

Reply:
Not needed anymoresince we removed the plugins